snapcraft/internal/project_loader/_config.py

```diff
@@ -121,7 +121,7 @@ def __init__(self, project_options=None):
         # This is required for patching to work correctly
         if _requires_patchelf_snap(self.data['confinement']):
             # TODO use stable once possible
-            self.build_snaps.add('patchelf/edge')
+            self.build_snaps.add('patchelf/latest/edge')
```

Comment:
Why this?

Reply:
Otherwise the test failed like this:

    "/tmp/autopkgtest.6TVtmK/build.46t/snapcraft/snapcraft/internal/repo/snaps.py", line 119, in is_classic
        return store_channels[self.channel]['confinement'] == 'classic'
    KeyError: 'edge'

Which from my understanding is because of "tracking":

    $ snap info patchelf | grep tracking
    tracking:    latest/edge

The same is true for me locally, not just on Travis.

Reply:
Also, this is the result of `_get_store_channels()`:

    {'latest/edge': {'epoch': '0', 'channel': 'edge', 'version': '0.9-36-g29c085f', 'revision': '6', 'confinement': 'classic', 'size': 327680}}